src/pages/links-tips-picks/index.js

```diff
@@ -0,0 +1,107 @@
+import {PropTypes} from 'react'
+import {StyleSheet} from 'aphrodite'
+import {upToBig} from '<styles>/media-queries'
+import Page from '<components>/page'
+import HeaderBar from '<components>/header'
+import {markdownToHTML} from '<utils>/utils'
+
+function htmlifyLinksPicksAndTips(thing) {
+  const stripP = true
+  thing.linksHTML = thing.links.map(l => markdownToHTML(l, stripP))
+  thing.tipsHTML = thing.tips.map(t => markdownToHTML(t, stripP))
+  thing.picksHTML = thing.picks.map(p => markdownToHTML(p, stripP))
+}
+
+function htmlifyGroup(people) {
+  return people.map(person => {
```

Comment:
By returning `thing` from `htmlifyLinksPicksAndTips`, this could be as simple as:

```javascript
function htmlifyGroup(people) {
  return people.map(htmlifyLinksPicksAndTips)
}
```